lib/evernote/api/client.py

```diff
@@ -3,8 +3,8 @@
 import inspect
 import re
 import oauth2 as oauth
-import urllib
-import urlparse
+import urllib.request, urllib.parse, urllib.error
+import urllib.parse
```

Comment:
I think this could be removed since you're importing `urllib.parse` in the previous line.